spec/api_spec.rb

```diff
@@ -31,6 +71,27 @@
       })
     end
 
+    it 'builds user_attributes_hash with devices' do
+      user = users.first
+      user[:devices] = devices
+      expect(api.send(:build_user_attributes_hash, users.first)).to eq({
+                                                                         userId: first_user_id,
```

Comment:
re: style - this is _way_ out of style guide compliance.  max line length is 120.

basically you just need to left indent this stuff, or assign it to a variable beforehand.  sorry i can overlook a little whitespace but i can't overlook this.
https://github.com/bbatsov/ruby-style-guide

Reply:
Please feel free to make the style work for you. :)

Reply:
the reason the style guide exists is so that questions like this don't come up.  it is the arbiter, and this is just not how you style ruby.  there's an [entire company/application](https://houndci.com/) devoted to rejecting this type of formatting.

are you willing to let your otherwise well composed PR die on this molehill?

Reply:
i'll make you a deal - you make this block sane and i'll handle the merge of `build_user_attributes`

Reply:
done.